### ISSUE TITLE
UTime Resetting on my Server.

### ISSUE THREAD

**Issue by axis1234** [TeamUlysses/ulx#172]
### Steps to reproduce
1. Have a server installed with ULib, UTime, and ULX.
2. Install the addons through FTP and files. (not workshop).
3. Start the server, and see everything (hopefully) function properly, or experience this very same issue.
![dsdadsadsaadssadasdasdasdsad](https://user-images.githubusercontent.com/69921077/90693265-2121cd80-e23c-11ea-8d23-0e843fee9344.PNG)

### Expected behavior
Time is tracked and is kept to be shown on our scoreboard. `sui_scoreboard_v2`

### Actual behavior
Hours are reset after a server restart, and/or inherit from another user online.

### Error(s) in server console, if any
none seen.

### Error(s) in player's console, if any
none seen.
### Version
ULib v2.63
ULX v3.73

**Comment by Nayruden** [comment]
UTime saves data into the SQLite database, so it could be a file system permission issue. Are you having problems with any other save data?

**Comment by axis1234** [comment]
nope, most of my stuff goes through data files IN the server, but UTime has been the only issue.
it was working just fine until a recent host switch of the server.
